cmd/checks.go

```diff
@@ -13,6 +13,9 @@ const (
 	statusWarning
 	statusFailure
 	statusUnknown
+
+	// exhibitor admin router port
+	exhibitorPort = 8181
```

Comment:
Yup we will move all these constants to the runner once https://github.com/dcos/dcos-checks/pull/9 gets merged